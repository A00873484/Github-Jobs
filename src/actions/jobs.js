export const RECEIVE_JOBS = 'RECEIVE_JOBS'

export function receiveJobs (jobs) {
  return {
    type: RECEIVE_JOBS,
    jobs
  }
}
