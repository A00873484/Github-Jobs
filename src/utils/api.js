import appendToGet from './appendToGet'
import axios from 'axios'

export async function searchForJobs ({ description, fulltime, location }) {
  const res = await axios.get(`http://localhost:3001/github/jobs${appendToGet({ description, location, full_time: fulltime })}`)
  return res.data
}

export async function searchForJob (id) {
  const res = await axios.get(`http://localhost:3001/github/job?id=${id}`)
  return res.data
}
