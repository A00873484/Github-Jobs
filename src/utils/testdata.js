export const DataObject = {    
  "3c5fc02a-159b-4930-92aa-16bbc436f98a": {
    company: "Snickfish",
    company_logo: "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…ca60bf3947b4d117de93ba82e7b031fb2dc/snickfish-g-small.fw.png",
    company_url: "http://Snickfish",
    created_at: "Sat Feb 20 12:38:44 UTC 2021",
    description: "<p>LOOKING FOR LOCAL PHP DEVELOPER IN THE KNOXVILLE/MARYVILLE, TENNESSEE AREA.\nWe are a title management company based in Maryville, TN.  We service over 41 states across the U.S. and are looking for someone with an upbeat professional attitude who's eager to hang with us.  It's an exciting and fun place to work where you can learn, earn, and grow with us! This position is NOT a remote position. Please do NOT apply if you cannot work at our offices in Maryville, Tennessee.</p>\n<p>PHP DEVELOPER\nQualified, self-driven developer responsible for maintaining, updating, and upgrading client and server-side HTML /CSS /JS, PHP &amp; Node Systems.\nResponsibilities include setting deadlines, performing or automating regular maintenance tasks, performing system account updates, and optimizing existing systems in both PHP 5.6 and 7.2.\nPay is relevant to work experience and knowledge base.</p>\n<p>General Requirements\nExperienced in Large or Scalable Web Development\nSeasoned in PHP\nWork Well in a fast-paced environment\nMust be able to use high-level logic &amp; reasoning\nMust have excellent communication skill\nMust be able to plan and set deadlines effectively\nMust be willing to learn, and keep up to date knowledge on web development practices\nKnowledgeable in Standard HTML, CSS &amp; JS\nKnowledgeable in Objective, Functional, &amp; Procedural Programing &amp; Methodologies\nAWS Experience (including but not limited to EC2, Route 53,  EBS operational &amp; maintenance experience)\nMySQL General Experience\nDocker &amp; Containerization knowledge, and experience.\n5 + years general programing experience\n2 + years of Object-Oriented programing experience\nLanguage/ Framework /Library Requirements:\nPHP 5.6\nPHP 7.3\nNodeJS (JavaScript)\nMySQL\nComposer (PHP)\nNPM (NodeJS)\njQuery (JavaScript)\nBootstrap (HTML, CSS)\nRecommended Frameworks/Libraries\nLaravel (PHP)\nReact (NodeJS)\nEnvironment Familiarity (Required)\nDocker\nAWS EC2\nLinux</p>\n<p>We provide a nice friendly environment with professional people.\nWe hire for life.\nSome staffers have been with us since our conception 13 years ago. Our employees arrive to work happy and leave happy and we like that a lot.\nThe dress code is casual.\nYou will receive vacation pay, major holiday pay, PTO, and bonuses.</p>\n<p>Hours are 8-5 Monday-Friday, with evenings and weekends off.\nSend us your resume and tell us a little about yourself, and we'll contact you with further details about our company and how to schedule an interview.</p>\n",
    how_to_apply: "<p>Email your resume to <a href=\"mailto:NH8570@gmail.com\">NH8570@gmail.com</a></p>\n",
    id: "3c5fc02a-159b-4930-92aa-16bbc436f98a",
    location: "Maryville, Tennessee",
    title: "PHP Developer - NOT A REMOTE POSITION",
    type: "Full Time",
    url: "https://jobs.github.com/positions/3c5fc02a-159b-4930-92aa-16bbc436f98a"
  }, 
  "07d9afe8-0916-40f5-a508-2a42d9dc6c98": {
    company: "brightwheel ",
    company_logo: "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…lkIn19--8665ccd64baddd8e0766413744ed285dbe693bf1/Logo%20.png",
    company_url: "https://mybrightwheel.com/careers/",
    created_at: "Mon Feb 22 16:45:45 UTC 2021",
    description: "<p>Our Mission and Opportunity\nEarly education is one of the greatest determinants of childhood outcomes, is a must for working families, and has lasting social and economic impact. Brightwheel’s mission is to transform early education for every teacher, child, and parent — by giving teachers meaningfully more time with students each day, engaging parents in the development of their kids, and supporting the small businesses that make up the backbone of the $100 billion early education market. Brightwheel is the most loved technology brand in early education globally, trusted by thousands of educators and millions of families.</p>\n<p>Our Team\nWe are a distributed team, with offices in San Francisco, Denver, and Austin along with fully remote team members. Our team is passionate, talented, and customer-focused. Our exceptional investor group includes Addition, Bessemer Venture Partners, Chan Zuckerberg Initiative, GGV Capital, Lowercase Capital, Emerson Collective, and Mark Cuban.</p>\n<p>We believe that everyone—from our employees to the students, teachers, and administrators we serve— should be given the opportunity to learn and thrive, whatever their background may be. We celebrate diversity in all forms because it allows our team and the communities we serve to reach their full potential and do their best work.</p>\n<p>Who You Are\nWe are seeking a senior software engineer who is not only well-rounded technically, but also someone who is passionate about the positive impact their code can make in people's lives. Brightwheel engineers work closely with product and design, as well as with other areas of the business to deliver a platform that meets our teachers’ &amp; parents’ biggest needs. Every engineer at brightwheel is afforded a high degree of ownership and autonomy. We are a lean, fast-moving team and value individuals who can optimize for being impactful while considering both technical and business drivers.</p>\n<p>What You'll Do\n-Work alongside product and design to scope and implement new features or product lines.\n-Be a steward of quality, scalability, security, and performance. You'll work with other engineers to ensure that we have a solid foundation that serves our customers, and enables the team to continue building a great product.\n-Design and architect new software systems, or introduce new technologies, as appropriate to support brightwheel’s growth.\n-Share knowledge and expertise across the engineering team</p>\n<p>Qualifications, Skills, &amp; Abilities\n-4+ years of professional experience, preferably in dynamic companies/teams\n-Expertise in one or more application programming languages (e.g Ruby, JavaScript or Python)\n-Proficiency in building and debugging distributed systems on AWS\n-Proficiency in both contributing to and designing APIs</p>\n<p>Candidates with varying backgrounds are encouraged to apply. We do not require prior knowledge of technologies in our stack, only a desire to learn them. At brightwheel you will encounter:</p>\n<ul>\n<li>React.js</li>\n<li>Ruby on Rails</li>\n<li>PostgreSQL</li>\n<li>Redis</li>\n<li>CircleCI</li>\n<li>Kubernetes</li>\n<li>AWS managed services, like Redshift and ECS</li>\n</ul>\n<p>Brightwheel is proud to celebrate diversity and is committed to building an inclusive workplace regardless of race, color, religion, sex (including pregnancy and gender identity), national origin, political affiliation, sexual orientation, marital status, disability, genetic information, age, membership in an employee organization, retaliation, parental status, military service, or other non-merit factor.</p>\n",
    how_to_apply: "<p>Please apply using the link below:</p>\n<p><a href=\"https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7…ba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub\">https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7eba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub</a></p>\n",
    id: "07d9afe8-0916-40f5-a508-2a42d9dc6c98",
    location: "Anywhere in the US",
    title: "Senior Software Engineer, Backend",
    type: "Full Time",
    url: "https://jobs.github.com/positions/07d9afe8-0916-40f5-a508-2a42d9dc6c98",
  }
}
export const DataArray = [
  {
    company: "Snickfish",
    company_logo: "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…ca60bf3947b4d117de93ba82e7b031fb2dc/snickfish-g-small.fw.png",
    company_url: "http://Snickfish",
    created_at: "Sat Feb 20 12:38:44 UTC 2021",
    description: "<p>LOOKING FOR LOCAL PHP DEVELOPER IN THE KNOXVILLE/MARYVILLE, TENNESSEE AREA.\nWe are a title management company based in Maryville, TN.  We service over 41 states across the U.S. and are looking for someone with an upbeat professional attitude who's eager to hang with us.  It's an exciting and fun place to work where you can learn, earn, and grow with us! This position is NOT a remote position. Please do NOT apply if you cannot work at our offices in Maryville, Tennessee.</p>\n<p>PHP DEVELOPER\nQualified, self-driven developer responsible for maintaining, updating, and upgrading client and server-side HTML /CSS /JS, PHP &amp; Node Systems.\nResponsibilities include setting deadlines, performing or automating regular maintenance tasks, performing system account updates, and optimizing existing systems in both PHP 5.6 and 7.2.\nPay is relevant to work experience and knowledge base.</p>\n<p>General Requirements\nExperienced in Large or Scalable Web Development\nSeasoned in PHP\nWork Well in a fast-paced environment\nMust be able to use high-level logic &amp; reasoning\nMust have excellent communication skill\nMust be able to plan and set deadlines effectively\nMust be willing to learn, and keep up to date knowledge on web development practices\nKnowledgeable in Standard HTML, CSS &amp; JS\nKnowledgeable in Objective, Functional, &amp; Procedural Programing &amp; Methodologies\nAWS Experience (including but not limited to EC2, Route 53,  EBS operational &amp; maintenance experience)\nMySQL General Experience\nDocker &amp; Containerization knowledge, and experience.\n5 + years general programing experience\n2 + years of Object-Oriented programing experience\nLanguage/ Framework /Library Requirements:\nPHP 5.6\nPHP 7.3\nNodeJS (JavaScript)\nMySQL\nComposer (PHP)\nNPM (NodeJS)\njQuery (JavaScript)\nBootstrap (HTML, CSS)\nRecommended Frameworks/Libraries\nLaravel (PHP)\nReact (NodeJS)\nEnvironment Familiarity (Required)\nDocker\nAWS EC2\nLinux</p>\n<p>We provide a nice friendly environment with professional people.\nWe hire for life.\nSome staffers have been with us since our conception 13 years ago. Our employees arrive to work happy and leave happy and we like that a lot.\nThe dress code is casual.\nYou will receive vacation pay, major holiday pay, PTO, and bonuses.</p>\n<p>Hours are 8-5 Monday-Friday, with evenings and weekends off.\nSend us your resume and tell us a little about yourself, and we'll contact you with further details about our company and how to schedule an interview.</p>\n",
    how_to_apply: "<p>Email your resume to <a href=\"mailto:NH8570@gmail.com\">NH8570@gmail.com</a></p>\n",
    id: "3c5fc02a-159b-4930-92aa-16bbc436f98a",
    location: "Maryville, Tennessee",
    title: "PHP Developer - NOT A REMOTE POSITION",
    type: "Full Time",
    url: "https://jobs.github.com/positions/3c5fc02a-159b-4930-92aa-16bbc436f98a",
  },
  {
    company: "brightwheel ",
    company_logo: "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…lkIn19--8665ccd64baddd8e0766413744ed285dbe693bf1/Logo%20.png",
    company_url: "https://mybrightwheel.com/careers/",
    created_at: "Mon Feb 22 16:45:45 UTC 2021",
    description: "<p>Our Mission and Opportunity\nEarly education is one of the greatest determinants of childhood outcomes, is a must for working families, and has lasting social and economic impact. Brightwheel’s mission is to transform early education for every teacher, child, and parent — by giving teachers meaningfully more time with students each day, engaging parents in the development of their kids, and supporting the small businesses that make up the backbone of the $100 billion early education market. Brightwheel is the most loved technology brand in early education globally, trusted by thousands of educators and millions of families.</p>\n<p>Our Team\nWe are a distributed team, with offices in San Francisco, Denver, and Austin along with fully remote team members. Our team is passionate, talented, and customer-focused. Our exceptional investor group includes Addition, Bessemer Venture Partners, Chan Zuckerberg Initiative, GGV Capital, Lowercase Capital, Emerson Collective, and Mark Cuban.</p>\n<p>We believe that everyone—from our employees to the students, teachers, and administrators we serve— should be given the opportunity to learn and thrive, whatever their background may be. We celebrate diversity in all forms because it allows our team and the communities we serve to reach their full potential and do their best work.</p>\n<p>Who You Are\nWe are seeking a senior software engineer who is not only well-rounded technically, but also someone who is passionate about the positive impact their code can make in people's lives. Brightwheel engineers work closely with product and design, as well as with other areas of the business to deliver a platform that meets our teachers’ &amp; parents’ biggest needs. Every engineer at brightwheel is afforded a high degree of ownership and autonomy. We are a lean, fast-moving team and value individuals who can optimize for being impactful while considering both technical and business drivers.</p>\n<p>What You'll Do\n-Work alongside product and design to scope and implement new features or product lines.\n-Be a steward of quality, scalability, security, and performance. You'll work with other engineers to ensure that we have a solid foundation that serves our customers, and enables the team to continue building a great product.\n-Design and architect new software systems, or introduce new technologies, as appropriate to support brightwheel’s growth.\n-Share knowledge and expertise across the engineering team</p>\n<p>Qualifications, Skills, &amp; Abilities\n-4+ years of professional experience, preferably in dynamic companies/teams\n-Expertise in one or more application programming languages (e.g Ruby, JavaScript or Python)\n-Proficiency in building and debugging distributed systems on AWS\n-Proficiency in both contributing to and designing APIs</p>\n<p>Candidates with varying backgrounds are encouraged to apply. We do not require prior knowledge of technologies in our stack, only a desire to learn them. At brightwheel you will encounter:</p>\n<ul>\n<li>React.js</li>\n<li>Ruby on Rails</li>\n<li>PostgreSQL</li>\n<li>Redis</li>\n<li>CircleCI</li>\n<li>Kubernetes</li>\n<li>AWS managed services, like Redshift and ECS</li>\n</ul>\n<p>Brightwheel is proud to celebrate diversity and is committed to building an inclusive workplace regardless of race, color, religion, sex (including pregnancy and gender identity), national origin, political affiliation, sexual orientation, marital status, disability, genetic information, age, membership in an employee organization, retaliation, parental status, military service, or other non-merit factor.</p>\n",
    how_to_apply: "<p>Please apply using the link below:</p>\n<p><a href=\"https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7…ba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub\">https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7eba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub</a></p>\n",
    id: "07d9afe8-0916-40f5-a508-2a42d9dc6c98",
    location: "Anywhere in the US",
    title: "Senior Software Engineer, Backend",
    type: "Full Time",
    url: "https://jobs.github.com/positions/07d9afe8-0916-40f5-a508-2a42d9dc6c98",
  }
]
export const DataInstance = DataObject["3c5fc02a-159b-4930-92aa-16bbc436f98a"]
