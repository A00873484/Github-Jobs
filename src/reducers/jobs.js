import { RECEIVE_JOBS } from '../actions/jobs'

export default function jobs (state = {}, action) {
  switch (action.type) {
    case RECEIVE_JOBS:
      return action.jobs
    default:
      return state
  }
}
