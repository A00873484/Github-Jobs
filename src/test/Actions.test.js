import configureStore from 'redux-mock-store'
import { DataArray } from '../utils/testdata'
import { receiveJobs, RECEIVE_JOBS } from '../actions/jobs'

const mockStore = configureStore()
const store = mockStore()

describe('selectAvatar', () => {
    test('Dispatches the correct action and payload', () => {
      const expectedActions = [
        {
            type: RECEIVE_JOBS,
            jobs: DataArray
        }
      ]
      store.dispatch(receiveJobs(DataArray))
      expect(store.getActions()).toEqual(expectedActions)
      expect(store.getActions()).toMatchSnapshot();
    })
})