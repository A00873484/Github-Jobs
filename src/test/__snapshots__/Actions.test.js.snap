// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`selectAvatar Dispatches the correct action and payload 1`] = `
Array [
  Object {
    "jobs": Array [
      Object {
        "company": "Snickfish",
        "company_logo": "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…ca60bf3947b4d117de93ba82e7b031fb2dc/snickfish-g-small.fw.png",
        "company_url": "http://Snickfish",
        "created_at": "Sat Feb 20 12:38:44 UTC 2021",
        "description": "<p>LOOKING FOR LOCAL PHP DEVELOPER IN THE KNOXVILLE/MARYVILLE, TENNESSEE AREA.
We are a title management company based in Maryville, TN.  We service over 41 states across the U.S. and are looking for someone with an upbeat professional attitude who's eager to hang with us.  It's an exciting and fun place to work where you can learn, earn, and grow with us! This position is NOT a remote position. Please do NOT apply if you cannot work at our offices in Maryville, Tennessee.</p>
<p>PHP DEVELOPER
Qualified, self-driven developer responsible for maintaining, updating, and upgrading client and server-side HTML /CSS /JS, PHP &amp; Node Systems.
Responsibilities include setting deadlines, performing or automating regular maintenance tasks, performing system account updates, and optimizing existing systems in both PHP 5.6 and 7.2.
Pay is relevant to work experience and knowledge base.</p>
<p>General Requirements
Experienced in Large or Scalable Web Development
Seasoned in PHP
Work Well in a fast-paced environment
Must be able to use high-level logic &amp; reasoning
Must have excellent communication skill
Must be able to plan and set deadlines effectively
Must be willing to learn, and keep up to date knowledge on web development practices
Knowledgeable in Standard HTML, CSS &amp; JS
Knowledgeable in Objective, Functional, &amp; Procedural Programing &amp; Methodologies
AWS Experience (including but not limited to EC2, Route 53,  EBS operational &amp; maintenance experience)
MySQL General Experience
Docker &amp; Containerization knowledge, and experience.
5 + years general programing experience
2 + years of Object-Oriented programing experience
Language/ Framework /Library Requirements:
PHP 5.6
PHP 7.3
NodeJS (JavaScript)
MySQL
Composer (PHP)
NPM (NodeJS)
jQuery (JavaScript)
Bootstrap (HTML, CSS)
Recommended Frameworks/Libraries
Laravel (PHP)
React (NodeJS)
Environment Familiarity (Required)
Docker
AWS EC2
Linux</p>
<p>We provide a nice friendly environment with professional people.
We hire for life.
Some staffers have been with us since our conception 13 years ago. Our employees arrive to work happy and leave happy and we like that a lot.
The dress code is casual.
You will receive vacation pay, major holiday pay, PTO, and bonuses.</p>
<p>Hours are 8-5 Monday-Friday, with evenings and weekends off.
Send us your resume and tell us a little about yourself, and we'll contact you with further details about our company and how to schedule an interview.</p>
",
        "how_to_apply": "<p>Email your resume to <a href=\\"mailto:NH8570@gmail.com\\">NH8570@gmail.com</a></p>
",
        "id": "3c5fc02a-159b-4930-92aa-16bbc436f98a",
        "location": "Maryville, Tennessee",
        "title": "PHP Developer - NOT A REMOTE POSITION",
        "type": "Full Time",
        "url": "https://jobs.github.com/positions/3c5fc02a-159b-4930-92aa-16bbc436f98a",
      },
      Object {
        "company": "brightwheel ",
        "company_logo": "https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpb…lkIn19--8665ccd64baddd8e0766413744ed285dbe693bf1/Logo%20.png",
        "company_url": "https://mybrightwheel.com/careers/",
        "created_at": "Mon Feb 22 16:45:45 UTC 2021",
        "description": "<p>Our Mission and Opportunity
Early education is one of the greatest determinants of childhood outcomes, is a must for working families, and has lasting social and economic impact. Brightwheel’s mission is to transform early education for every teacher, child, and parent — by giving teachers meaningfully more time with students each day, engaging parents in the development of their kids, and supporting the small businesses that make up the backbone of the $100 billion early education market. Brightwheel is the most loved technology brand in early education globally, trusted by thousands of educators and millions of families.</p>
<p>Our Team
We are a distributed team, with offices in San Francisco, Denver, and Austin along with fully remote team members. Our team is passionate, talented, and customer-focused. Our exceptional investor group includes Addition, Bessemer Venture Partners, Chan Zuckerberg Initiative, GGV Capital, Lowercase Capital, Emerson Collective, and Mark Cuban.</p>
<p>We believe that everyone—from our employees to the students, teachers, and administrators we serve— should be given the opportunity to learn and thrive, whatever their background may be. We celebrate diversity in all forms because it allows our team and the communities we serve to reach their full potential and do their best work.</p>
<p>Who You Are
We are seeking a senior software engineer who is not only well-rounded technically, but also someone who is passionate about the positive impact their code can make in people's lives. Brightwheel engineers work closely with product and design, as well as with other areas of the business to deliver a platform that meets our teachers’ &amp; parents’ biggest needs. Every engineer at brightwheel is afforded a high degree of ownership and autonomy. We are a lean, fast-moving team and value individuals who can optimize for being impactful while considering both technical and business drivers.</p>
<p>What You'll Do
-Work alongside product and design to scope and implement new features or product lines.
-Be a steward of quality, scalability, security, and performance. You'll work with other engineers to ensure that we have a solid foundation that serves our customers, and enables the team to continue building a great product.
-Design and architect new software systems, or introduce new technologies, as appropriate to support brightwheel’s growth.
-Share knowledge and expertise across the engineering team</p>
<p>Qualifications, Skills, &amp; Abilities
-4+ years of professional experience, preferably in dynamic companies/teams
-Expertise in one or more application programming languages (e.g Ruby, JavaScript or Python)
-Proficiency in building and debugging distributed systems on AWS
-Proficiency in both contributing to and designing APIs</p>
<p>Candidates with varying backgrounds are encouraged to apply. We do not require prior knowledge of technologies in our stack, only a desire to learn them. At brightwheel you will encounter:</p>
<ul>
<li>React.js</li>
<li>Ruby on Rails</li>
<li>PostgreSQL</li>
<li>Redis</li>
<li>CircleCI</li>
<li>Kubernetes</li>
<li>AWS managed services, like Redshift and ECS</li>
</ul>
<p>Brightwheel is proud to celebrate diversity and is committed to building an inclusive workplace regardless of race, color, religion, sex (including pregnancy and gender identity), national origin, political affiliation, sexual orientation, marital status, disability, genetic information, age, membership in an employee organization, retaliation, parental status, military service, or other non-merit factor.</p>
",
        "how_to_apply": "<p>Please apply using the link below:</p>
<p><a href=\\"https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7…ba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub\\">https://jobs.lever.co/brightwheel/6e96774f-8350-495d-a567-c7eba55f171d?lever-origin=applied&amp;lever-source%5B%5D=GitHub</a></p>
",
        "id": "07d9afe8-0916-40f5-a508-2a42d9dc6c98",
        "location": "Anywhere in the US",
        "title": "Senior Software Engineer, Backend",
        "type": "Full Time",
        "url": "https://jobs.github.com/positions/07d9afe8-0916-40f5-a508-2a42d9dc6c98",
      },
    ],
    "type": "RECEIVE_JOBS",
  },
]
`;
